x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/detail/index.tsx

```diff
@@ -216,7 +207,7 @@ export function Detail() {
     pkgName,
     pkgVersion,
     {
-      prerelease: prereleaseIntegrationsEnabled,
+      prerelease: true,
```

Comment:
A [couple of lines below](https://github.com/elastic/kibana/blob/60ffb369f0ee9154c7aacce04db431f961ef9b9d/x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/detail/index.tsx#L245-L247) we do again the same call (`prerelease` is also `true` in that case). Maybe we can try to avoid an extra call here?

Reply:
good point, removed the second call

Reply:
Thanks!